modules/swagger-jaxrs2/src/main/java/io/swagger/v3/jaxrs2/DefaultParameterExtension.java

```diff
@@ -191,7 +193,7 @@ private boolean handleAdditionalAnnotation(List<Parameter> parameters, Annotatio
                         paramType = getter.getType();
                     }
 
-                    for (final Annotation fieldAnnotation : getter.annotations()) {
+                    for (final Annotation fieldAnnotation : annotations(getter.getAllAnnotations())) {
```

Comment:
see also general review, we would keep as is to maintain support for jackson 2.8.x